Spigot-Server-Patches/0334-Configurable-speed-for-water-flowing-over-lava.patch

```diff
@@ -0,0 +1,65 @@
+From 35c565d267a562633607766e1264ac0cd4c01d87 Mon Sep 17 00:00:00 2001
```

Comment:
Please break this patch out into a separate PR so they can be reviewed independently.

There are issues with this patch (not your changes, the previous version of it was flawed) that we need to fix before bringing it back to 1.13